src/appdev/ApplicationRouter.js

```diff
@@ -102,7 +102,12 @@ class ApplicationRouter<T> {
     return async (req: Request, res: Response, next: NextFunction) => {
       try {
         const content = await this.content(req);
-        LogUtils.log({ path: this.getPath(), query: req.query });
+        const epochTime = Date.now();
+        const formattedTime = new Date(epochTime).toUTCString();
+        const message = {
+          epochTime, formattedTime, path: this.getPath(), query: req.query,
+        };
+        LogUtils.log(message);
```

Comment:
why not:

```
LogUtils.log({ 
  path: this.getPath(),
  query: req.query,
  ...
});
```

I don't think there's a point of declaring a variable just to use it in the next line!